themes/math_research_center/patterns/templates/node-simple/scss/node-simple.scss

```diff
@@ -1,2 +1,32 @@
 @charset "UTF-8";
 
+///
+/// SIMPLE NODE STYLES
+///
+
+// Decanter
+@import
+  'decanter-no-markup';
+
+// Stanford Math Research Center
+@import
+  '../utilities/mixins/mixins';
+
+.simple-node {
+  .field-s-visitor-photo {
```

Comment:
This is a node specific field. only the classes in the node-simple.html.twig should be used here.